GoogleForCreators/web-stories-wp
# Autosaves: Set up initial infrastructure

## Context

* Add feature flag (`improvedAutosaves` or something)
* In `AutoSaveHandler`, use `autoSave` instead of `saveStory` if flag is enabled



## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Visit experiments 
2. Note that an option for Improved Autosaves exists
3. Ensure saving a still works

## Reviews

### Does this PR have a security-related impact?

no



### Does this PR change what data or activity we track or use?

no



### Does this PR have a legal-related impact?

no



## Checklist




---



Fixes #12094
